src/thunderbots/software/ai/passing/pass.h

```diff
@@ -39,6 +39,13 @@ namespace AI::Passing
          */
         Point receiverPoint() const;
 
+        /**
+         * Gets the angle the receiver should be facing to receive the pass
+         *
+         * @return The angle the receiver should be facing to receive the pass
+         */
+        Angle receiverAngle() const;
```

Comment:
`receiverOrientation` might be slightly more descriptive here.

Reply:
renamed